project/backend/doc-server/src/main/resources/files/dd398660-ca95-4b58-97c8-5c7c5d2a8a31.py

```
@@ -0,0 +1,128 @@
+
+def generate_embeddings(chunks):
+    tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+    model = BertModel.from_pretrained('bert-base-uncased')
+    embeddings = []
+    for chunk in chunks:
+        inputs = tokenizer(chunk, return_tensors='pt', truncation=True, max_length=512, padding='max_length')
+        outputs = model(**inputs)
+        embeddings.append(outputs.last_hidden_state.mean(dim=1).detach().numpy())
+    return embeddings
+
+def save_to_faiss(context_id, embeddings, chunks):
```

Ruff (0.8.2)

Missing return type annotation for public function `save_to_faiss`

Add return type annotation: `None`

(ANN201)